node/src/reactor/joiner.rs

```diff
@@ -425,6 +457,30 @@ impl<R: Rng + CryptoRng + ?Sized> reactor::Reactor<R> for Reactor<R> {
                     };
                     self.dispatch_event(effect_builder, rng, Event::BlockByHeightFetcher(event))
                 }
+                Message::GetResponse {
+                    tag: Tag::Deploy,
+                    serialized_item,
+                } => {
+                    let deploy = match rmp_serde::from_read_ref(&serialized_item) {
+                        Ok(deploy) => Box::new(deploy),
+                        Err(err) => {
+                            error!("failed to decode deploy from {}: {}", sender, err);
+                            return Effects::new();
+                        }
+                    };
+                    let event = Event::DeployAcceptor(deploy_acceptor::Event::Accept {
```

Comment:
(Not this PR, but: `Event::Accept` should probably be called something like `Validate`, should it?)